.github/benchmark-files/xdebug-benchmark.ini

```diff
@@ -0,0 +1,3 @@
+zend_extension=xdebug.so
+xdebug.start_with_request=no
+xdebug.max_nesting_level=2048
```

Comment:
We need this because some of the functions in bench.php do deep recursion and the default nesting level is not enough